app/helpers/view_components/mentoring/inbox.rb

```diff
@@ -1,15 +1,24 @@
 module ViewComponents
   module Mentoring
     class Inbox < ViewComponent
-      initialize_with :conversations_request, :tracks_request, :sort_options
+      initialize_with :conversations_request, :tracks_request
 
       def to_s
-        react_component("mentoring-inbox", {
-                          conversations_request: conversations_request,
-                          tracks_request: tracks_request,
-                          sort_options: sort_options
-                        })
+        react_component(
+          "mentoring-inbox",
+          {
+            conversations_request: conversations_request,
+            tracks_request: tracks_request,
+            sort_options: SORT_OPTIONS
+          }
+        )
       end
+
+      SORT_OPTIONS = [
+        { value: 'recent', label: 'Sort by Most Recent' },
+        { value: 'exercise', label: 'Sort by Exercise' },
+        { value: 'student', label: 'Sort by Student' }
+      ].freeze
```

Comment:
I initially set this to be a `private_constant` but then used it in the tests so removed that.